src/app/containers/CpsRecommendations/SplitRecommendations/index.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { arrayOf, shape, number, string, func } from 'prop-types';
+import { storyItem } from '#models/propTypes/storyItem';
+import CpsRecommendations from '..';
+
+/*
+ * OPTIMIZELY: 003_hindi_experiment_feature.
+ * This component should be removed after the experiment.
+ */
+const SplitRecommendations = ({ items, ...props }) => {
+  const { parentColumns, part, splitRecsViewEventTracker } = props;
```

Comment:
I am not entirely sure what parentColumns is used for. It seems a useless prop to pass but I might be wrong?